docker-compose/camunda-8.6/README.md

```diff
@@ -152,31 +156,27 @@ If you enabled authentication for GRPC requests on Zeebe you need to provide cli
 
 ## Web Modeler Self-Managed
 
-> :information_source: Web Modeler Self-Managed is available to Camunda enterprise customers only.
+> :information_source: Web Modeler Self-Managed images with version <8.6.0-alpha2 are available to Camunda enterprise customers only. Follow the instructions in the root folder [README](../../README.md) to set up the environment for these versions.
```

Comment:
❓ Do we really need this paragraph? Instead, we could highlight the limitations regarding the license key.

Reply:
We should have a paragraph for users who still want to set up the old images and I want to redirect them to the right place. But I should still show the limitations of the developer license. I will add that

Reply:
For my understanding: Why would you try to set up the old images with the `8.6` version of the docker-compose files?

Reply:
I rather thought: someone now wants to set up 8.5 in general but is still in the 8.6 folder and just changes the values in the `.env`. But I think this is not likely. Instead they would switch folder or branch. I will remove the part.